src/css/theming.css

```
@@ -36,7 +36,7 @@
   --ifm-navbar-link-hover-color: var(--primary-font-color);
   --ifm-link-hover-decoration: none;
   --ifm-hover-overlay: var(--indigo-700);
-  --ifm-color-primary: var(--indigo-600);
+  --ifm-color-primary: #25c2a0;
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--ifm-color-primary is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)